### ISSUE TITLE
Game crashes when leaving world [BUG]

### ISSUE THREAD

**Issue by sharko789** [haubna/PhysicsMod#467]
**Describe the bug**
I just updated to the new version and now my game crashes every time I leave a world or server (the server doesn't have the mod installed).

**Stacktrace**
```
java.lang.NullPointerException: Cannot invoke "net.minecraft.class_638.method_8624(net.minecraft.class_2902$class_2903, int, int)" because "level" is null
	at net.diebuddies.minecraft.weather.WindSoundInstance.method_16896(WindSoundInstance.java:65)
	at net.minecraft.class_1140.method_4857(class_1140.java:254)
	at net.minecraft.class_1140.method_20185(class_1140.java:238)
	at net.minecraft.class_1144.method_18670(class_1144.java:249)
	at net.minecraft.class_310.method_1574(class_310.java:1861)
	at net.minecraft.class_310.method_1523(class_310.java:1135)
	at net.minecraft.class_310.method_1514(class_310.java:768)
	at net.minecraft.client.main.Main.method_44604(Main.java:244)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
```

Minecraft Version: 1.19.2
Physics Mod Version: 2.9.1

**Comment by haubna** [comment]
Will be fixed with 2.9.2, thanks for the report!

**Comment by haubna** [comment]
2.9.2 released with the fix.
